src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.request.UserRequestMapper;
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class AuthenticationController {
+    private final UserService userService;
+    private final UserRequestMapper userRequestMapper;
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist#:~:text=Use%20DtoRequestMapper%20and%20DtoResponseMapper%20as%20types%20when%20you%20declare%20mapper%20fields%20in%20controllers.

Reply:
the same for other mappers